tests/logs/delete/test_delete/test_delete_not_AmiPushItem.txt

```diff
@@ -0,0 +1,5 @@
+[    INFO] Loading items from pub:https://fakepub.com?task-id=12345
+[ WARNING] Push Item vhd_pushitem at None is not an AmiPushItem, dropping it from the queue.
+[ WARNING] Push Item vhd_pushitem at None is not an AmiPushItem, dropping it from the queue.
+[    INFO] Collecting results
+[    INFO] Delete completed
```

Comment:
Here as well it should fail since it didn't process anything IMHO

Reply:
I think I got it all